### DISCUSSION TITLE
hacktoberfest

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#35897] [Q&A] [answered]
Why is it only pull requests from bug squad members that are being tagged hacktoberfest?

**Comment by brianteeman** [comment]
it is also completely unnecessary

**Comment by chmst** [comment]
As you say, it is not necessary. The community members got an information in the community channel that they can ask for a label on their PRs - we don't know who joined the hacktoberfest.

**Reply by brianteeman** [reply]
It is not needed for Hacktoberfest and please remember that only a small % of the community have access to the glip channel I assume you refer to.

**Comment by brianteeman** [comment, marked as answer]
If the pr are not merged before the deadline they wont count anyway
